src/hotspot/share/runtime/globals.hpp

```diff
@@ -238,6 +238,9 @@ const int ObjectAlignmentInBytes = 8;
   product(bool, UseBASE64Intrinsics, false,                                 \
           "Use intrinsics for java.util.Base64")                            \
                                                                             \
+  product(bool, UsePolyIntrinsics, false,                                   \
```

Comment:
I'm not a fan of introducing new flags for individual intrinsics (there's already `-XX:DisableIntrinsic=_name` specifically for that), but since we already have many, shouldn't it be declared as a diagnostic flag, at least?

Reply:
Started removing the option, but its quite convenient to have the boolean global, so just made the option diagnostic. 
"done"